### PR TITLE
fix(ci): ci dashboard issue

### LINKED ISSUE #15161: Report successful AGW build to `magma-ci.web.app`
- The `AGW` build entry on `https://magma-ci.web.app/` is empty. The Make job that provides this information was removed in https://github.com/magma/magma/pull/15144. 
Screenshot from 2023-03-27 10-19-52

- To provide this information the `Bazel package job` in the `bazel.yml` workflow needs to be updated to provide the relevant information.
  - https://github.com/magma/magma/pull/15144/files#diff-71a71e993dac5ebcaa1ad57402f04ee226b14121a8052e4f994706670fd65a1a can be used as a starting point on what information is needed. 

Related to:
- https://github.com/magma/magma/issues/15159
- https://github.com/magma/magma/pull/15158

### PR DESCRIPTION
**Pull request by panyogesh**
Changes:
  1. Triggered AGW build push from build_all
  2. Related changes in Bazel

Test:
  1. Using github actions

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by maxhbr** [comment]
The changes look reasonable, lest see if it successfully posts the AGW build. Good work! Will review, once CI is done.

**Comment by maxhbr** [comment]
this is expected to close https://github.com/magma/magma/issues/15161

File: .github/workflows/build_all.yml
Comment (maxhbr) [inline .github/workflows/build_all.yml:111]: Tha bazel jobs seem to not be kicked off correctly:

**Review by maxhbr — CHANGES_REQUESTED** [review]
This does not look like it is triggering the bazel builds.

File: .github/workflows/build_all.yml
Comment (tapasmishra) [reply .github/workflows/build_all.yml:111]: > Tha bazel jobs seem to not be kicked off correctly:
> 
> 

This will use the bazel.yml from master branch. The current file in Master branch is not a reusable workflow yet. Once the PR will be merged to the Master branch it will start working as expected.

Comment (maxhbr) [reply .github/workflows/build_all.yml:111]: this was already tested? do you have links to runs in a fork where it worked or something like that?

Comment (maxhbr) [reply .github/workflows/build_all.yml:111]: @tapasmishra : If you have already understood that it is good and have seen that it works, you could give the approval here?

Comment (tapasmishra) [reply .github/workflows/build_all.yml:111]: @maxhbr You can check this repo where I have a simple scenario on this issue : https://github.com/tapaswl/test_workflow/tree/main

Comment (maxhbr) [reply .github/workflows/build_all.yml:111]: Ok, but then we can either force-merge this or first merge the changes to make bazel job callable with an independen PR first. Otherwise the merging will be blocked.

Comment (tapasmishra) [reply .github/workflows/build_all.yml:111]: I think force-merge will help to save duplicate works.

**Comment by maxhbr** [comment]
this is blocked by https://github.com/magma/magma/pull/15215, see comment by @panyogesh in https://magmacore.slack.com/archives/C01Q1T14YJ2/p1686906618507489?thread_ts=1686905954.161159&cid=C01Q1T14YJ2

**Review by maxhbr — APPROVED** [review]
Code looks sensible and @tapasmishra stated that it was tested and is working

**Pull request merged**
